core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -227,7 +229,7 @@ class LogCleaner(val config: CleanerConfig,
      * Clean a log if there is a dirty log available, otherwise sleep for a bit
      */
     private def cleanOrSleep() {
-      val cleaned = cleanerManager.grabFilthiestCompactedLog() match {
+      val cleaned = cleanerManager.grabFilthiestLog(time) match {
```

Comment:
Is there a reason to rename the method? The current name seems more precise since the log cleaner can now do both compaction and deletion.